requirements.txt

```diff
@@ -1,3 +1,3 @@
 requests==2.25.0
-cuenca-validations==0.6.9
+cuenca-validations==0.7.1.dev0
```

Comment:
@cuencandres @kerycdiaz we don't merge PRs that depend on pre-release libraries

Reply:
Aun no se libera cuenca-validations, estan haciendo pruebas de todo

Reply:
Then this PR can't be merged and released--until the code it depends on is also released

Reply:
ya quedo